triage/labels.rst

```diff
@@ -10,405 +10,135 @@ Many labels are shared for both use cases, while some are dedicated
-+---------------+------------------------------------------------------------+
-
-Linked pull requests
---------------------
-A link might be added manually using the cog icon next to this field.
-Most commonly though, if the PR includes "Fixes #xxx" in its description,
-the link will be added automatically.
-
-Generating Special Links in a Comment
-=====================================
-Using the following abbreviations in a comment will automatically generate
-a link to relevant web pages.
-
-+-------------------------------------------------------------+-------------------------------------------------------+
-| Comment abbreviation                                        | Description                                           |
-+=============================================================+=======================================================+
-| ``#<number>``,                                              | Links to the tracker issue or PR ``<number>`` (they   |
-| ``GH-<number>``                                             | share the same sequence of integers on GitHub).       |
-+-------------------------------------------------------------+-------------------------------------------------------+
-| ``BPO-<number>``                                            | Links to the old bug tracker at bugs.python.org.      |
-+-------------------------------------------------------------+-------------------------------------------------------+
-| a 10-, 11-, 12-, or 40-digit hex ``<number>``               | Indicates a Git changeset identifier and              |
-|                                                             | generates a link to changeset ``<number>`` on GitHub. |
-+-------------------------------------------------------------+-------------------------------------------------------+
-
-.. _Doc: https://github.com/python/cpython/tree/main/Doc/
-.. _Grammar: https://github.com/python/cpython/tree/main/Grammar/
-.. _Lib: https://github.com/python/cpython/tree/main/Lib/
-.. _Lib/doctest.py: https://github.com/python/cpython/blob/main/Lib/doctest.py
-.. _Lib/test: https://github.com/python/cpython/tree/main/Lib/test/
-.. _Lib/test/regrtest.py: https://github.com/python/cpython/blob/main/Lib/test/regrtest.py
-.. _Lib/test/support: https://github.com/python/cpython/tree/main/Lib/test/support/
-.. _Lib/unittest: https://github.com/python/cpython/tree/main/Lib/unittest/
-.. _Modules: https://github.com/python/cpython/tree/main/Modules/
-.. _Objects: https://github.com/python/cpython/tree/main/Objects/
-.. _Parser: https://github.com/python/cpython/tree/main/Parser/
-.. _Python: https://github.com/python/cpython/tree/main/Python/
+The following labels only apply to PRs.  They are either set automatically
+by bots, or added by humans to trigger specific bot behaviors.
+
+* :gh-label:`DO-NOT-MERGE`: for PRs that shouldn't be merged in their current
+  state.  It also prevents ``miss-islington`` from being able to automatically
```

Comment:
Not sure if we need code formatting on `miss-islington`

It's written "miss-islington" or "[@miss-islington](https://github.com/miss-islington)" at https://github.com/python/miss-islington

and "Miss Islington (bot)" at https://github.com/python/miss-islington

Shall we also link to the repo or user for one or more of these?

Reply:
Yeah, I'd suggest

```suggestion
  state.  It also prevents `miss-islington <https://github.com/python/miss-islington>`
  from being able to automatically
```

Reply:
This was implemented in the other comment, which reflowed it as well